blockchain/processing.py

```diff
@@ -166,6 +173,17 @@ def trigger_handler():
 
         elif config[PHASE] == 5:
             self._add_registration(5, self._execute_phase_5, config)
+            self._add_registration("timestamp", self._execute_timestamping, config)
+
+            # Setup phase 5 cron
+            trigger = CronTrigger(second='*/30')
```

Comment:
Might have to explain the config override. I can change but I don't fully understand what is being asked.